src/components/interactive-builder/edit-question-modal.component.tsx

```diff
@@ -397,6 +422,40 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
               </div>
             )}
 
+            {questionToEdit.type === 'personAttribute' && (
+              <div>
+                <FormLabel className={styles.label}>
+                  {t('searchForBackingPersonAttributeType', 'Search for a backing person attribute type')}
+                </FormLabel>
+                {personAttributeNameLookupError ? (
+                  <InlineNotification
+                    kind="error"
+                    lowContrast
+                    className={styles.error}
+                    title={t('errorFetchingPersonAttributeTypes', 'Error fetching  person attribute types')}
+                    subtitle={t('pleaseTryAgain', 'Please try again.')}
+                  />
+                ) : null}
+                {isLoadingPersonAttributeName ? (
+                  <InlineLoading className={styles.loader} description={t('loading', 'Loading') + '...'} />
+                ) : (
+                  <ComboBox
+                    id="personAttributeTypeLookup"
+                    items={personAttributeTypes}
+                    itemToString={(item: PersonAttributeType) => item?.display}
+                    onChange={({ selectedItem }: { selectedItem: PersonAttributeType }) => {
+                      handleAttributeTypeSelect(selectedItem);
+                    }}
+                    placeholder={t('choosePersonAttributeType', 'Choose a  person attribute type')}
+                    initialSelectedItem={personAttributeTypes.find(
+                      (personAttributeType) =>
+                        personAttributeType?.uuid === questionToEdit.questionOptions?.attributeType,
+                    )}
+                  />
+                )}
+              </div>
+            )}
+
             {fieldType !== 'ui-select-extended' && questionToEdit.type !== 'patientIdentifier' && (
```

Comment:
```suggestion
            {fieldType !== 'ui-select-extended' &&
              questionToEdit.type !== 'patientIdentifier' &&
              questionToEdit.type !== 'personAttribute' && (
```

Reply:
So we don't see the concept lookup field for `personAttribute` type questions.

Reply:
oh yeah, thanks @denniskigen ..
had forgotten to push the commit for this